data/libs_map.yml

```diff
@@ -0,0 +1,373 @@
+- libpmem: pmem_check_version.3 pmem_deep_drain.3 pmem_deep_flush.3 pmem_deep_persist.3 pmem_drain.3 pmem_errormsg.3 pmem_flush.3 pmem_has_auto_flush.3 pmem_has_hw_drain.3 pmem_is_pmem.3 pmem_map_file.3 pmem_memcpy.3 pmem_memcpy_nodrain.3 pmem_memcpy_persist.3 pmem_memmove.3 pmem_memmove_nodrain.3 pmem_memmove_persist.3 pmem_memset.3 pmem_memset_nodrain.3 pmem_memset_persist.3 pmem_msync.3 pmem_persist.3 pmem_unmap.3
```

Comment:
We have to enable again/update scripts to autogenerate documentation on  this repo and auto genarate this yml (create  PR using pmdk bot account).
(I think changes on pmdk repo are needed)